lib/select/index.js

```diff
@@ -17,10 +17,7 @@ export default class OCSelect extends React.Component {
 
     this.state = {
       active: false,
-      selection:
```

Comment:
Prior to the changes, `state.selection` wouldn't render `props.selection` if there was no instance of it in `props.options`. Can you ensure that this functionality is retained?